web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -45,11 +45,14 @@ export function SendForm({ ...props }) {
 
   function handlePercent(amount = 0, percent: number) {
     let newAmount = amount * percent;
+    const feesAsNumber = Number(fees);
+    if (newAmount > balance - feesAsNumber)
+      newAmount = Math.max(balance - feesAsNumber, 0);
```

Comment:
```suggestion
    const newAmount = amount * percent;
    const feesAsNumber = Number(fees);
    
    if (newAmount > balance - feesAsNumber)
      newAmount = Math.max(balance - feesAsNumber, 0);
```

Do we realy need this Math.max!?

Reply:
to avoid negative number, do you have another idea ?